alaska/tests/test_parser.py

```diff
@@ -42,6 +64,50 @@ def test_parse():  # 1000080059
     assert result == ({"depth": ["DEPT"], "gamma ray": ["GR"]}, [])
 
 
+def test_parse_2():
+    """
+    Test that Aliaser can parse las file with an empty mnemonic
+    """
+    aliaser = Alias()
+    # import pdb; pdb.set_trace()
```

Comment:
```suggestion
```

Reply:
Good point.  Okay the suggested change is on the new commit b2ad07f.